### PR TITLE
Translate ITE-6 to AsciiDoc

### PR DESCRIPTION
**Pull request by adityasaky**
https://github.com/in-toto/ITE/blob/master/ITE/1/README.adoc#ite-file-format requires ITEs to be in AsciiDoc. This PR translates ITE-6 to adoc in preparation for fixing up the ITE for acceptance.

Also, note: a link has been updated from pointing to ITE-5 to DSSE as ITE-5 is no longer just about DSSE.

### CODE REVIEW

File: ITE/6/README.adoc [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+link:https://github.com/in-toto/attestation/blob/main/spec/predicates/spdx.md[spdx]
+for supporting link:https://spdx.dev[SPDX], and
+link:https://github.com/in-toto/attestation/blob/main/spec/predicates/link.md[link]
+as a drop-in replacement for the existing link schema.
+
+A future ITE will update the layout format to take advantage of this extra
+information.
+
+More broadly, this ITE moves in-toto to follow the
+link:https://github.com/slsa-framework/slsa/blob/main/controls/attestations.md[SLSA attestation model]
```
Comment (MarkLodato) [inline ITE/6/README.adoc:58]: optional: You can replace the link with https://slsa.dev/attestation-model

File: ITE/6/README.adoc [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+*   Build a larger ecosystem around software artifact metadata that is not
+    specific to in-toto (or Binary Authorization or any other consumer). By
+    supporting a more generic, extensible attestation format, we allow CI/CD
+    pipelines, vulnerability scanners, and other systems to generate a single
+    set of attestations that can be consumed by anyone. Furthermore, this paves
+    the way for in-toto to support existing formats such as SPDX or CycloneDX.
+
+Furthermore, this ITE fits within the
+link:https://github.com/slsa-framework/slsa[SLSA framework]. The provenance
```
Comment (MarkLodato) [inline ITE/6/README.adoc:145]: optional: https://slsa.dev

**Review by MarkLodato — APPROVED** [review]

File: ITE/6/README.adoc (hunk 1 above)
Comment (adityasaky) [reply ITE/6/README.adoc]: Done.

File: ITE/6/README.adoc (hunk 2 above)
Comment (adityasaky) [reply ITE/6/README.adoc]: Done.

**Review by trishankatdatadog — APPROVED** [review]

**Review by SantiagoTorres — APPROVED** [review]

**Pull request merged**